### PR TITLE
Sct 12 schedule single meeting

### PR DESCRIPTION
**Pull request by JaycobDuffel**

### CODE REVIEW

File: scutl.py [hunk 1]
```diff
@@ -1,7 +1,11 @@
 import os
 import requests
+from slack_bolt import App
+from slack_sdk.errors import SlackApiError
 from pathlib import Path
 from dotenv import load_dotenv
+import requests
+from datetime import datetime
```
Comment (katiemcgoff) [inline scutl.py:9]: We've got some repeat imports here.

File: scutl.py [hunk 2]
```diff
@@ -67,35 +71,328 @@ def update_home_tab(client, event, logger):
...
+                    }
+                ],
+            },
+        )
+        logger.info(result)
+
+    except SlackApiError as e:
+        logger.error("Error creating conversation: {}".format(e))
+
+date_time = {}
```
Comment (katiemcgoff) [inline scutl.py:370]: Looks like this is a global variable, ie: used in a few places. I think we should move it to the top of the file so it doesn't get lost and is more clearly a global variable.

File: scutl.py [hunk 3]
```diff
@@ -67,35 +71,328 @@ def update_home_tab(client, event, logger):
...
+                        "initial_date": datetime.today().strftime('%Y-%m-%d'),
+                        "placeholder": {
+                          "type": "plain_text",
+                          "text": "Select a date"
+                        }
+                      },
+                    },
+                    {
+                      "type": "section",
+                      "block_id": "section678",
```
Comment (katiemcgoff) [inline scutl.py:132]: Do we know what these block_id labels represent? I see hardcoded values, but what do they mean? Should they be hardcoded or calculated?

Comment (JaycobDuffel) [reply scutl.py]: I think they are hardcoded typically but don't actually need them in this case so I will remove them!

**Review by katiemcgoff — APPROVED** [review]

**Pull request merged**